traffic-sniffer/lib/request-metadata/sniffer-manager/mock-manager-controller.ts

```diff
@@ -0,0 +1,180 @@
+import { Express, NextFunction, Request, Response } from "express";
+import { SnifferManager } from "./sniffer-manager";
+import { MockNotFoundError } from "../sniffer/mock/exceptions";
+
+export class MockManagerController {
```

Comment:
Also, what if we separate out routes and controllers?

Reply:
How do you suggest we do that?

Reply:
Use router as separate bit using express routers

Reply:
We can do this separately.. will it up separately